gcp_variant_transforms/vcf_to_bq_common.py

```diff
@@ -84,6 +85,17 @@ def get_pipeline_mode(known_args):
   return PipelineModes.SMALL
 
 
+def form_absolute_file_name(directory, job_name, file_name):
```

Comment:
since this is no longer a 'common' function, please move it back to vcf_to_bq :) you can leave it inline as it was before... It's kinda an odd function, so I prefer to not expose it too much.

Reply:
Done.